CHANGES.md

```diff
@@ -1,5 +1,17 @@
 # Changes
 
+## edge-23.7.1
+
+This edge release adds support for the upstream `gateway.networking.k8s.io`
+HTTPRoute resource (rather than the `policy.linkerd.io` CRD installed by
```

Comment:
I think this should say "in addition to" instead of "rather than".  The `policy.linkerd.io` HttpRoute is still supported.

Reply:
good call, changed it!